xml/System/Random.xml

```diff
@@ -371,7 +371,7 @@ Random.NextDouble() * (maxValue - minValue) + minValue
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The default seed value is derived from the system clock and has finite resolution. As a result, different <xref:System.Random> objects that are created in close succession by a call to the parameterless constructor will have identical default seed values and, therefore, will produce identical sets of random numbers. This problem can be avoided by using a single <xref:System.Random> object to generate all random numbers. You can also work around it by modifying the seed value returned by the system clock and then explicitly providing this new seed value to the <xref:System.Random.%23ctor%28System.Int32%29> constructor. For more information, see the <xref:System.Random.%23ctor%28System.Int32%29> constructor.  
+ The default seed value is derived from the system clock and has finite resolution. As a result, on the .NET Framework only, different <xref:System.Random> objects that are created in close succession by a call to the parameterless constructor will have identical default seed values and, therefore, will produce identical sets of random numbers. This problem can be avoided by using a single <xref:System.Random> object to generate all random numbers. You can also work around it by calling this constructor on .NET Core, or by modifying the seed value returned by the system clock and then explicitly providing this new seed value to the <xref:System.Random.%23ctor%28System.Int32%29> constructor. For more information, see the <xref:System.Random.%23ctor%28System.Int32%29> constructor.  
```

Comment:
> You can also work around it by calling this constructor on .NET Core […]

This doesn't make much sense to me. If you're suggesting that the workaround is switching to .Net Core, then I think that should be phrased more clearly, but I'm not sure it's actually reasonable. People are not going to switch to .Net Core just to avoid few extra lines of code.

> You can also work around it […] by modifying the seed value returned by the system clock

This text wasn't changed by this PR, but I find it hard to understand and I think it should be clarified too: the clock returns time, not a seed value and there's no way to modify the value that's returned by the clock.